### PR TITLE
Added net45 TFM to Redis project

### PR DESCRIPTION
**Pull request by kichalla**
@Tratcher

### CODE REVIEW

File: src/Microsoft.Framework.Caching.Redis/project.json [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
         "Microsoft.Framework.OptionsModel": "1.0.0-*",
         "StackExchange.Redis": "1.0.333"
     },
-  "frameworks": {
-    "dnx451": {}
-  }
-}
+    "frameworks": {
+        "dnx451": { },
+        "net45": { }
```
Comment (Tratcher) [inline src/Microsoft.Framework.Caching.Redis/project.json]: Make it net451. we don't support net45 anywhere in the stack.

Comment (kichalla) [inline src/Microsoft.Framework.Caching.Redis/project.json]: I referred this https://github.com/aspnet/Caching/blob/dev/src/Microsoft.Framework.Caching.Memory/project.json#L15 ...should it change too?

**Comment by davidfowl** [comment]
What's this for EF?

**Comment by kichalla** [comment]
@davidfowl no, its related to issue https://github.com/aspnet/Caching/issues/87

File: src/Microsoft.Framework.Caching.Redis/project.json (hunk 1 above)
Comment (Tratcher) [inline src/Microsoft.Framework.Caching.Redis/project.json]: yes.

Comment (kichalla) [inline src/Microsoft.Framework.Caching.Redis/project.json]: @Tratcher updated the PR.

**Comment by Tratcher** [comment]
Hmm, you need to review this with EF: https://github.com/aspnet/EntityFramework/blob/dev/src/EntityFramework.Core/project.json#L13-L29

**Comment by PinpointTownes** [comment]
:ship: :it:

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**